sqrl-tools/sqrl-config/src/main/java/com/datasqrl/config/ConnectorConfigImpl.java

```diff
@@ -49,8 +48,7 @@ public TableType getTableType() {
     String connectorName = getConnectorName().get().toLowerCase();
     TableType tableType = CONNECTOR_TYPE_MAP.get(connectorName);
     if (tableType == null) {
-      log
-          .warn("Unrecognized connector type: {}. Defaulting to STREAM table for import.",
+      log.info("Unrecognized connector type: {}. Defaulting to STREAM table for import.",
```

Comment:
That's why I changed the loglvl from warn to info, because I also thought it would look like there's an issue. Nevertheless I reduced the lvl to debug and changed the message slightly. Let me know if it is good.